metaflow/plugins/cards/card_decorator.py

```diff
@@ -44,16 +44,23 @@ def _load_card_package(self):
             if len(metaflow_cards.__path__._path) > 0:
                 metaflow_cards_root = metaflow_cards.__path__._path[0]
 
+        from . import card_modules
+
+        card_modules_root = os.path.dirname(card_modules.__file__)
+
         if metaflow_cards_root:
             # What if a file is too large and
             # gets tagged along the metaflow_cards
             # path; In such cases we can have huge tarballs
             # that get created;
             # Should we have package suffixes added over here?
             for path_tuple in self._walk(metaflow_cards_root):
-                # print(path_tuple)
                 yield path_tuple
 
+        for path_tuple in self._walk(card_modules_root):
```

Comment:
Unsure this is needed. Isn't the whole metaflow code base included anyways? Is this because there are more suffixes? If that's the case, you can just add those as suffixes. There is an var that does that already.

Reply:
Can you point to that variable?

Reply:
METAFLOW_EXTENSIONS_PACKAGE_SUFFIXES

Reply:
But this variable requires stuff to be present in the `self.metaflow_extensions_root` from which it will allow the `METAFLOW_EXTENSIONS_PACKAGE_SUFFIXES`; Currently we don't have that.